community/communities.md

```diff
@@ -89,3 +89,21 @@ Link: [https://stackoverflow.com/questions/tagged/machine-translation](https://s
 The **Machine-Translation** tag on Quora gathers user questions and answers related to machine translation, including its development and performance.
 
 Link: [https://www.quora.com/topic/Machine-Translation/](https://www.quora.com/topic/Machine-Translation/)
+
+
+## Communities
+
+### MaCoCu
+
+**MaCoCu** focuses on collecting monolingual and parallel data from the Internet, specially for under-resourced languages and DSI-specific data.
```

Comment:
I'd use blockquote and just directly quote them (with curly quotes) rather than copying.